src/modules/item/types.ts

```diff
@@ -146,6 +146,7 @@ export type Item = BaseItem & {
   inCatalyst: boolean
   contents: Record<string, string>
   contentHash: string | null
+  serverContentHash: string | null
```

Comment:
I was thinking we might be able to name this better. Both here and the server. **I wouldn't change it here**, but I feel like neither local nor server signify the real use of this prop.
Maybe cached_content_hash?

Reply:
It's definitely a difficult property to name. I wouldn't go with `cached_content_hash`, because it's the content hash of the entity that's stored in the last save to the server. We could rename both content hashed to:
`blockchainContentHash` and `actualContentHash` or `currentContentHash`, what do you think about this?